packages/core/src/modules/slack/use-cases/send-stackbot-mass-notification.ts

```diff
@@ -0,0 +1,19 @@
+import { type GetBullJobData } from '@/infrastructure/bull/bull.types';
+import { job } from '@/infrastructure/bull/use-cases/job';
+import { db } from '@/infrastructure/database';
+
+export async function sendStackBotMassNotification(
+  data: GetBullJobData<'slack.mass.message'>
+) {
+  const members = await db.selectFrom('students').select(['slackId']).execute();
```

Comment:
I know we're doing the conditional check for `member.slackId` within the for loop, but I think an additional improvement would be adding a `.where('slackId', 'is not', null)` check in the SQL query itself!